review/tests.py

```diff
@@ -51,3 +51,21 @@ def test_delete_related_dogowner(self, test_review):
         dogowner.delete()
         assert dogowner not in DogOwner.objects.all()
         assert test_review in Review.objects.all()
+
+    def test_get_review_by_daycare_id(self):
+        daycare = DayCare.create(email="review@gmail.com", username="review", password="pass", name="reviewDayCare",
+                                 description="This is test daycare for review", price_per_day=10, capacity=50,
+                                 area="Merkaz", city="Tel-Aviv", address="The best review")
+        reviews = Review.get_review_by_daycare_id(daycare.id)
+        assert len(reviews) == 0
+
+        Review.create(review='review 1', rating=1, daycare_id=daycare.id, dogowner_id=1)
+
+        reviews = Review.get_review_by_daycare_id(daycare.id)
+        assert len(reviews) == 1
```

Comment:
Here len is more appropriate, but I'll add another test to check we got the Review we expected (the same one from Review.create)

Reply:
Added